README.md

```diff
@@ -28,7 +28,7 @@ by <a href="https://kennethreitz.org/">Kenneth Reitz</a> & is protected by The <
 
 <p>&nbsp;</p>
 
-```pycon
+```python3
```

Comment:
This is unrelated and unnecessary as part of this PR

Reply:
Reverted @sigmavirus24